packaging/dbscripts/upgrade/pre_upgrade/0000_config.sql

```diff
@@ -131,7 +131,7 @@ select fn_db_add_config_value('FenceStartStatusDelayBetweenRetriesInSec','60','g
 select fn_db_add_config_value('FenceStartStatusRetries','3','general');
 select fn_db_add_config_value('FenceStopStatusDelayBetweenRetriesInSec','60','general');
 select fn_db_add_config_value('FenceStopStatusRetries','3','general');
-select fn_db_add_config_value('FencePowerWaitParam','apc=power_wait,apc_snmp=power_wait,bladecenter=power_wait,cisco_ucs=power_wait,drac5=power_wait,drac7=power_wait,eps=delay,hpblade=power_wait,ilo=power_wait,ilo2=power_wait,ilo3=power_wait,ilo4=power_wait,ipmilan=power_wait,redfish=power_wait,rsa=power_wait,rsb=power_wait,wti=power_wait','general');
+select fn_db_add_config_value('FencePowerWaitParam','apc=power_wait,apc_snmp=power_wait,amt_ws=power_wait,bladecenter=power_wait,cisco_ucs=power_wait,drac5=power_wait,drac7=power_wait,eps=delay,hpblade=power_wait,ilo=power_wait,ilo2=power_wait,ilo3=power_wait,ilo4=power_wait,ipmilan=power_wait,redfish=power_wait,rsa=power_wait,rsb=power_wait,wti=power_wait','general');
```

Comment:
Not related to the logic itself: looks like there is an attempt to keep in queries sorting by the column names?
If so, then the new `amt` should come before `apc` - actually to be the 1st one.